Client/Frontend/Browser/Playlist/Controllers/PlaylistFolderSharingManagementView.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveShared
+import BraveUI
+import DesignSystem
+
+struct PlaylistFolderSharingManagementView: View {
+  private struct UX {
+    static let hPadding = 20.0
+    static let vPadding = 20.0
+  }
+  
+  var onAddToPlaylistPressed: (() -> Void)?
+  var onSettingsPressed: (() -> Void)?
+  var onCancelPressed: (() -> Void)?
+  
+  var body: some View {
+    VStack {
+      Text(Strings.PlaylistFolderSharing.offlineManagementViewTitle)
+        .font(.title2)
+        .fontWeight(.medium)
+        .foregroundColor(Color(.bravePrimary))
+        .padding(.horizontal, UX.hPadding)
+        .padding(.bottom, UX.vPadding)
+      
+      Text(String(format: "%@\n\n%@", Strings.PlaylistFolderSharing.offlineManagementViewDescription, Strings.PlaylistFolderSharing.offlineManagementViewSubDescription))
```

Comment:
I put them together like this because it makes more sense than having a separate label with spacing. It ends up to be almost the same thing. I could make them all one string I guess with new lines in-between.